Automattic/wp-calypso
# Reader site stream spring clean

Clean up `reader/site-stream` to follow current coding guidelines.

No functional changes.

### To test

Visit a site stream (e.g. http://calypso.localhost:3000/discover) and ensure that there are no display issues or errors in the console.

Visit an empty site stream (e.g.  http://calypso.localhost:3000/read/blogs/12345) and make sure the empty content message displays correctly:

<img width="728" alt="screen shot 2016-11-17 at 13 09 16" src="https://cloud.githubusercontent.com/assets/17325/20371067/1710bd14-acc7-11e6-8b6c-e82f1806c275.png">
